manifests/r/RStudio/quarto/1.2.269/RStudio.quarto.locale.en-US.yaml

```diff
@@ -0,0 +1,47 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: RStudio.quarto
+PackageVersion: 1.2.269
+PackageLocale: en-US
+Publisher: Posit Software, PBC
+PublisherUrl: https://posit.co/
+PublisherSupportUrl: https://posit.co/support/
+PrivacyUrl: https://posit.co/about/privacy-policy/
+Author: J.J. Allaire, Charles Teague, Carlos Scheidegger, Yihui Xie, Christophe Dervieux
+PackageName: Quarto
+PackageUrl: https://quarto.org/
+License: GPL v2
+LicenseUrl: https://quarto.org/license.html
+Copyright: Copyright (C) Posit Software, PBC
+CopyrightUrl: https://quarto.org/trademark.html
+ShortDescription: Quarto® is an open-source scientific and technical publishing system built on Pandoc
+Description: |-
+  Main features of Quarto:
+     Create dynamic content with Python, R, Julia, and Observable.
+     Author documents as plain text markdown or Jupyter notebooks.
+     Publish high-quality articles, reports, presentations, websites, blogs, and books in HTML, PDF, MS Word, ePub, and more.
+     Author with scientific markdown, including equations, citations, crossrefs, figure panels, callouts, advanced layout, and more.
+Moniker: quarto
+Tags:
+- document
+- jupyter
+- markdown
+- ojs
+- pandoc
+- publishing
+- r
+- reproducibility
+- rmarkdown
+# Agreements:
+# ReleaseNotes:
+ReleaseNotesUrl: https://github.com/quarto-dev/quarto-cli/releases/tag/v1.2.269
```

Comment:
Release note URL to use is rather this one 
https://github.com/quarto-dev/quarto-cli/blob/main/news/changelog-1.2.md

v1.2.269 is the release of Quarto v1.2 and the github release page will contain only some notes from 1.2.267 and 1.2.269

Previous version in Winget use the news file instead. We should do the same here

Reply:
Would it be necessary to obtain the news file from the 1.2 tag rather than main?